app/views/layouts/homepage.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for(:navbar) do %>
   <div class="image-masthead">
-    <div class="background-container" style="background-image: url('<%= banner_image %>')"></div>
+    <div class="background-container" style="background-image: linear-gradient(rgba(255, 255, 255, 0.3),rgba(255, 255, 255, 0.3)), url('<%= banner_image %>'); background-size: cover"></div>
```

Comment:
Is there any ways to get this inline style out of the html?

Reply:
I can probably pull the background-size out, but the background-image is stuck